ui/app/models/totp-key.js

```diff
@@ -43,9 +62,7 @@ const validations = {
       level: 'warn',
     },
   ],
-  period: [{ type: 'number', message: 'Period must be a number.' }],
-
-  skew: [{ validator: (value) => SKEW.includes(value), message: 'Skew must be one of ' + SKEW.join(', ') }],
+  qr_size: [{ type: 'number', message: 'QR size must be a number' }],
 };
 
 @withModelValidations(validations)
```

Comment:
below, I don't think we need `withExpandedAttributes()`? since I think we're just using `withFormFields()`